_sass/_base.scss

```diff
@@ -0,0 +1,214 @@
+/**
```

Comment:
Where is that coming from? If you're importing this base from somewhere, I'd like to know where I can update it should browsers become incompatible with it at some point.